dev-python/transforms3d/transforms3d-0.3.1.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{8..10} )
```

Comment:
```
* Log file: /var/log/portage/build/dev-python/transforms3d-0.3.1:20210618-082918.log

* DISTUTILS_USE_SETUPTOOLS value is probably incorrect
*   have:     DISTUTILS_USE_SETUPTOOLS=bdepend (or unset)
*   expected: DISTUTILS_USE_SETUPTOOLS=no
```

Reply:
**Woops.** This is me mopping sweat from my forehead. That's obviously wrong, so I'll get that patched right up. Thanks for the last-minute save... *as always.* :+1: